.env.example

```diff
@@ -47,6 +47,8 @@ LOG_LEVEL=debug
 OCTANE_SERVER=swoole
 OCTANE_HTTPS=false
 
+#VITE_INERTIA_SSR_PORT=13714
```

Comment:
I expected this to be uncommented, but I guess it shouldn't really matter. It's hard to tell from looking at the rest of the file what should and shouldn't be commented out.